### DISCUSSION TITLE
Der 1200AX ist gefreetz

### DISCUSSION THREAD

**Discussion by freetz-ng-mod** [Freetz-NG/freetz-ng#490] [Allgemein]
![Bildschirmfoto vom 2022-03-10 21-17-59](https://user-images.githubusercontent.com/79326105/157747589-d117f862-817f-4027-af6f-58b8e4d5e7e6.png)
![Bildschirmfoto vom 2022-03-10 21-17-33](https://user-images.githubusercontent.com/79326105/157747592-2d7d8e6b-1f6b-419c-a633-34dcdc7e3fb4.png)
![Bildschirmfoto vom 2022-03-10 21-16-59](https://user-images.githubusercontent.com/79326105/157747593-3145a1f1-33ed-4ddf-b297-997b2b6481e3.png)
![Bildschirmfoto vom 2022-03-10 21-40-47](https://user-images.githubusercontent.com/79326105/157750819-fd055fe3-7a85-4939-a00f-aa9add1987d6.png)

./push_firmware 1200AX_07.30.all_freetz-ng-18958MO-b1d5bf05e_20220310-211321.image -cmd ftp -map 0x40000000 -ram 256

**Comment by fda77** [comment]
Schön wenn es auf Anhieb funktioniert! push firmware hab ich angepasst, es sollte jetzt ohne weitere Parameter funktionieren. Da war auch ein Typo, wegen dem der 6000 nicht ohne Parameter funktionierte
64763cdd4 a47cce927
Juis muss ich mir noch anschauen, evtl liegt das einem Script von PeterPawn

**Reply by freetz-ng-mod** [reply]
Ich habe das mit dem push nun auch viel besser verstanden, daher habe ich es ja versucht. Und sehe da es ging. 
Das ist super das push firmware angepasst hast auf den 1200AX, daher habe ich ja die Informationen mit gepostet wie es ging.
Das mit juis finde ich jetzt nicht so wichtig. Ich glaube das Prob mit dem Routern 7530AX ist interessanter.

**Comment by fda77** [comment]
Das blöde an der ram-größe und Speicheradressen ist dass es keinen (bekannten) Weg gibt es auszulesen und man es stattdessen manuell in das Script packen muss.
juis-check kannst du auch testen, das Problem sollte es auf jedem FIT gegeben haben: 207cf8220

**Comment by freetz-ng-mod** [comment]
Bin schon dabei ihn zu flashen ;-)
Man kommt nur über ein recover an die Adressen. Das gute ist, es gibt nur 7 Boxen zur Zeit
https://www.digital-eliteboard.com/threads/supportthread-wie-flashe-ich-ein-freetz-image.335766/post-4012612

Daher müsste es dann bei dem 4060 das sein 
-map 0x40000000 -ram 1024 

```
 * MAPSTART=0x40000000
 * FULLSIZE=0x40000000     (1024 MB)
 * MAPLIMIT=0x80000000
 * FILESIZE=0x01c70000     (~28 MB)
 * FREESIZE=0x3e390000     (~995 MB)
 * MTDSTART=0x7e390000
```

Edit


```
  | Please wait ...  
Local version: 268.07.30-93528 Using version: 268.07.29-92600  
Found newer version: 268.07.30  done.
-- | --


```
geht

**Reply by fda77** [reply]
Dort steht
```
SETENV memsize 0x3e300000
```
also 995mb. Das sollte man einfach so übernehmen, der Rest wird wohl für irgendwas reserviert

Bei 1gb (=0x40000000) und oberer Grenze von 0x80000000 müsste 0x80000000-0x40000000 = 0x40000000 passen

 -> 3a5598d21

**Reply by freetz-ng-mod** [reply]
Habe hippie2000 den dump geschickt. Der sagte 
die 4060 kann auch wie der 6000 behandelt werden, grad die dumps eingebaut.

https://boxmatrix.info/wiki/FIT-Image#EVA-Upload

**Reply by fda77** [reply]
Wird sich zeigen ob diese default Werte passen: 51fbc8fd9

**Comment by fda77** [comment]
Der 1200ax nutzt die Sourcen des 6000, damit sollten Kernelmodule funktionieren. Es ist bei beiden eh noch der ältere 4.4.60, nicht wie bei 7530ax. Wobei beide Repeater eh kein USB haben

**Reply by freetz-ng-mod** [reply]
Auf den Kernel habe ich nicht so geachtet. Dass sie kein USB haben, kann ich hier ja live sehen
